community/kernel/src/main/java/org/neo4j/kernel/impl/factory/GraphDatabaseFacade.java

```diff
@@ -197,7 +195,7 @@ public GraphDatabaseFacade()
     /**
      * Create a new Core API facade, backed by the given SPI.
      */
-    public void init( SPI spi, Config config )
+    public void initSPI( SPI spi, Config config )
```

Comment:
initSPI is a bit confusing imho, since it looks like we initialise SPI here, but actually we just provide facade with dependencies, so i would keep old name instead